jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/JettyLoggerFactory.java

```diff
@@ -166,4 +175,166 @@ public boolean setLoggerLevel(String loggerName, String levelName)
         jettyLogger.setLevel(level);
         return true;
     }
+
+    @Override
+    public Object getAttribute(String name) throws AttributeNotFoundException
+    {
+        Objects.requireNonNull(name, "Attribute Name");
+
+        switch (name)
+        {
+            case "LoggerNames":
```

Comment:
I'm afraid `loggerNames` is also a valid value, so unfortunately we need `if/else` and `equalsIgnoreCase()`.

Reply:
The name here and the name in MBeanInfo must match, case included.
You cannot access the MBean attribute in a case insensitive way.

Example:

``` java
MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
ObjectName objectName = ObjectName.getInstance("java.lang", "type", "OperatingSystem");
System.out.printf("OSVer (normal): %s%n", mbeanServer.getAttribute(objectName, "Version"));
System.out.printf("OSVer (lowercase): %s%n", mbeanServer.getAttribute(objectName, "version"));
```

results in ...

```
OSVer (normal): 5.8.0-7630-generic
Exception in thread "main" javax.management.AttributeNotFoundException: No such attribute: version
	at java.management/com.sun.jmx.mbeanserver.PerInterface.getAttribute(PerInterface.java:81)
	at java.management/com.sun.jmx.mbeanserver.MBeanSupport.getAttribute(MBeanSupport.java:206)
	at java.management/javax.management.StandardMBean.getAttribute(StandardMBean.java:372)
	at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.getAttribute(DefaultMBeanServerInterceptor.java:641)
	at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.getAttribute(JmxMBeanServer.java:678)
```